nixos/modules/virtualisation/containerd.nix

```
@@ -0,0 +1,91 @@
+# Systemd services for containerd.
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.virtualisation.containerd;
+
+in
+
+{
+  ###### interface
+
+  options.virtualisation.containerd = {
+    enable =
+      mkOption {
+      type = types.bool;
+      default = false;
+      description =
+      ''
+        This option enables containerd, a daemon that manages
+        linux containers.
+      '';
+    };
+
+    listenOptions =
+      mkOption {
+      type = types.listOf types.str;
+      default = ["/run/containerd/containerd.sock"];
+      description =
+      ''
+        A list of unix and tcp containerd should listen to. The format follows
```

A list of unix and tcp what? Sockets?

```suggestion
        A list of unix and TCP sockets containerd should listen to. The format follows
```